docs/docusaurus/docs/core/introduction/community_resources.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Community Resources'
```

Comment:
Should this be sentence cased (Community resources)?

Reply:
It should, thanks for the catch.